gvr-360photo/app/build.gradle

```diff
@@ -21,16 +30,17 @@ android {
         jni.srcDirs = [] // no auto generation of Android.mk
         jniLibs.srcDir 'libs' // pre-compiled libraries
     }
-
-    applicationVariants.all { variant ->
-        variant.outputs.each { output ->
-            output.outputFile = new File(
-                    output.outputFile.parent,
-                     rootProject.name + ".apk")
-        }
-    }
+
+    applicationVariants.all { variant ->
+        variant.outputs.each { output ->
+            output.outputFile = new File(
+                    output.outputFile.parent,
+                     rootProject.name + ".apk")
+        }
+    }
 }
 
 dependencies {
-    compile project(':framework')
+    compile(name:'framework-debug', ext:'aar')

```

Comment:
It should be a generic name until we have a way to pick all aars from a location.